src/server.cpp

```diff
@@ -364,62 +363,89 @@ static void sigint(int signo) {
 		ws->stop();
 }
 
+/* pretty print CLI options */
```

Comment:
I prefer libfmt to iostreams here, at least for consistency - you'd do something like

`fmt::print(stuff);`